src/lazyload-image.ts

```diff
@@ -62,10 +61,10 @@ function setLoadedStyle(element: HTMLElement) {
     return element;
 }
 
-export function lazyLoadImage(image: HTMLElement, imagePath: string, errorImgPath: string, offset: number) {
+export function lazyLoadImage(image: HTMLElement, imagePath: string, errorImgPath: string, offset: number, window) {
     return (scrollObservable: Observable<Event>) => {
         return scrollObservable
-            .filter(() => isVisible(image, offset))
+            .filter(() => isVisible(image, offset, window))
```

Comment:
As a comment above; it is confusing to have a `window` variable that shadowing the global variable with the same name.